antarest/launcher/service.py

```diff
@@ -35,8 +37,7 @@ def update(self, job_result: JobResult) -> None:
         self.repository.save(job_result)
 
     def run_study(self, study_uuid: str) -> UUID:
-        # TODO use true parameters from user request
-        params = StorageServiceParameters()
+        params = StorageServiceParameters(user=Auth.get_current_user())
```

Comment:
parameters were created here temporarily. Like for `storage`, parameters will be construct in `web` part. This will also avoid to use a patch in test.